rts/motoko-rts/src/types.rs

```diff
@@ -506,7 +558,9 @@ impl Obj {
     }
 
     pub unsafe fn as_blob(self: *mut Self) -> *mut Blob {
```

Comment:
I wonder whether it could make sense to use dedicated `as_blob` functions, depending on the context, e.g. for texts etc.

Reply:
I did a grep and many of the as_blob uses are immediatly following allocation with a known tag, so perhaps not that useful. Will take another look though